### PR TITLE
🌐 Add Polish translation for `docs/pl/docs/help-fastapi.md`

### PR DESCRIPTION
**Pull request by romabozhanovgithub**
Relates to https://github.com/tiangolo/fastapi/discussions/9195

This PR is a translation of docs/pl/docs/help-fastapi.md

This is my first PR to this repository. I would appreciate a code review. Please let me know if it has any problems.

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit 679878ab3fc9f77e34afe2a2ceea4b11f836c7b3 at: https://1e5494a5.fastapitiangolo.pages.dev

File: docs/pl/docs/help-fastapi.md [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+
+### Nie zadawaj pytań na czacie
+
+Miej na uwadze, że ponieważ czaty pozwalają na bardziej "swobodną rozmowę", łatwo jest zadawać pytania, które są zbyt ogólne i trudniejsze do odpowiedzi, więc możesz nie otrzymać odpowiedzi.
+
+Na GitHubie szablon poprowadzi Cię do napisania odpowiedniego pytania, dzięki czemu łatwiej uzyskasz dobrą odpowiedź, a nawet rozwiążesz problem samodzielnie, zanim zapytasz. A na GitHubie mogę się upewnić, że zawsze odpowiadam na wszystko, nawet jeśli zajmuje to trochę czasu. Osobiście nie mogę tego zrobić z systemami czatu. 😅
+
+Rozmowy w systemach czatu nie są też tak łatwo przeszukiwalne jak na GitHubie, więc pytania i odpowiedzi mogą zaginąć w rozmowie. A tylko te na GitHubie liczą się do zostania [Ekspertem FastAPI](fastapi-people.md#experts){.internal-link target=_blank}, więc najprawdopodobniej otrzymasz więcej uwagi na GitHubie.
+
+Z drugiej strony w systemach czatu jest tysiące użytkowników, więc jest duża szansa, że znajdziesz tam kogoś do rozmowy, prawie cały czas. 😄
```
Comment (isulim) [inline docs/pl/docs/help-fastapi.md:244]:
```suggestion
Z drugiej strony w systemach czatu są tysiące użytkowników, więc jest duża szansa, że znajdziesz tam kogoś do rozmowy, prawie w każdej chwili. 😄
```

**Review by isulim — CHANGES_REQUESTED** [review]
Most of translation is ok, but I have some suggestions.

Większość tłumaczenia jest ok, ale mam trochę propozycji.

**Comment by romabozhanovgithub** [comment]
@isulim Thanks for you review, really appreciate

**Comment by tiangolo** [comment]
📝 Docs preview for commit 026faf5b27f2a788b4dea303093ca91c30ed6ceb at: https://fc058b53.fastapitiangolo.pages.dev

**Review by isulim — APPROVED** [review]

**Comment by isulim** [comment]
Hello @tiangolo 
Why does smokeshow/coverage fail on file `tests/test_openapi_examples.py`?
I never changed this file.

**Comment by romabozhanovgithub** [comment]
> Hello @tiangolo Why does smokeshow/coverage fail on file `tests/test_openapi_examples.py`? I never changed this file.

Same here

**Comment by tiangolo** [comment]
It seems it was a line in a dummy test I left uncovered and I probably didn't notice while Smokeshow was having errors. But now I added it, and updating the branch here should solve it. 🤓

**Comment by tiangolo** [comment]
📝 Docs preview for commit 5c4fcabcdf9a38e4599f365a4fa0cf473c1c981f at: https://ae496c77.fastapitiangolo.pages.dev

File: docs/pl/docs/help-fastapi.md [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+
+---
+
+Oto, co warto mieć na uwadze i jak przeglądać pull request:
+
+### Zrozum problem
+
+* Najpierw upewnij się, że **rozumiesz problem**, który próbuje rozwiązać pull request. Może on mieć dłuższą dyskusję w Dyskusji na GitHubie lub w problemie.
+
+* Jest też duża szansa, że pull request nie jest tak naprawdę potrzebny, ponieważ problem można rozwiązać w **inny sposób**. Wtedy możesz to zasugerować lub o to zapytać.
```
Comment (bluefish6) [inline docs/pl/docs/help-fastapi.md:157]:
```suggestion
* Jest też duża szansa, że pull request nie jest konieczny, ponieważ problem można rozwiązać w **inny sposób**. Wtedy możesz to zasugerować lub o to zapytać.
```

File: docs/pl/docs/help-fastapi.md [hunk 3]
```diff
@@ -0,0 +1,265 @@
...
+
+### Nie martw się stylem
+
+* Nie martw się zbytnio rzeczami takimi jak style wiadomości commitów, scalę i połączę je ręcznie, dostosowując commit.
```
Comment (bluefish6) [inline docs/pl/docs/help-fastapi.md:161]:
```suggestion
* Nie przejmuj się zbytnio rzeczami takimi jak style wiadomości commitów, przy wcielaniu pull requesta łączę commity i modyfikuję opis sumarycznego commita ręcznie.
```

File: docs/pl/docs/help-fastapi.md [hunk 4]
```diff
@@ -0,0 +1,265 @@
...
+
+* Nie martw się również zasadami stylu, istnieją już zautomatyzowane narzędzia sprawdzające to.
```
Comment (bluefish6) [inline docs/pl/docs/help-fastapi.md:163]:
```suggestion
* Nie przejmuj się również stylem kodu, automatyczne narzędzia w repozytorium sprawdzają to samodzielnie.
```

File: docs/pl/docs/help-fastapi.md [hunk 5]
```diff
@@ -0,0 +1,265 @@
...
+
+A jeśli istnieje jakaś inna potrzeba dotycząca stylu lub spójności, poproszę bezpośrednio o to, lub dodam commity z potrzebnymi zmianami.
```
Comment (bluefish6) [inline docs/pl/docs/help-fastapi.md:165]:
```suggestion
A jeśli istnieje jakaś konkretna potrzeba dotycząca stylu lub spójności, sam poproszę o zmiany lub dodam commity z takimi zmianami.
```

File: docs/pl/docs/help-fastapi.md [hunk 6]
```diff
@@ -0,0 +1,265 @@
...
+
+### Sprawdź kod
+
+* Sprawdź i przeczytaj kod, zobacz, czy ma sens, **uruchom go lokalnie** i zobacz, czy faktycznie rozwiązuje problem.
```
Comment (bluefish6) [inline docs/pl/docs/help-fastapi.md:169]:
```suggestion
* Przeczytaj kod, zastanów się czy ma sens, **uruchom go lokalnie** i potwierdź czy faktycznie rozwiązuje problem.
```

File: docs/pl/docs/help-fastapi.md [hunk 7]
```diff
@@ -0,0 +1,265 @@
...
+
+* Następnie **skomentuj**, mówiąc, że to zrobiłeś, dzięki temu będę wiedział, że naprawdę to sprawdziłeś.
```
Comment (bluefish6) [inline docs/pl/docs/help-fastapi.md:171]:
```suggestion
* Następnie dodaj **komentarz** z informacją o tym, że sprawdziłeś kod, dzięki temu będę miał pewność, że faktycznie go sprawdziłeś.
```

File: docs/pl/docs/help-fastapi.md [hunk 8]
```diff
@@ -0,0 +1,265 @@
...
+
+!!! info
+    Niestety, nie mogę po prostu ufać PR-om, które mają kilka zatwierdzeń.
```
Comment (bluefish6) [inline docs/pl/docs/help-fastapi.md:174]:
```suggestion
    Niestety, nie mogę ślepo ufać PR-om, nawet jeśli mają kilka zatwierdzeń.
```

File: docs/pl/docs/help-fastapi.md [hunk 9]
```diff
@@ -0,0 +1,265 @@
...
+
+    Kilka razy zdarzyło się, że są PR-y z 3, 5 lub więcej zatwierdzeniami, prawdopodobnie dlatego, że opis jest interesujący, ale gdy sprawdzam PR-y, są one w rzeczywistości uszkodzone, mają błąd lub nie rozwiązują problemu, który twierdzą, że rozwiązują. 😅
```
Comment (bluefish6) [inline docs/pl/docs/help-fastapi.md:176]:
```suggestion
    Kilka razy zdarzyło się, że PR-y miały 3, 5 lub więcej zatwierdzeń (prawdopodobnie dlatego, że opis obiecuje rozwiązanie ważnego problemu), ale gdy sam sprawdziłem danego PR-a, okazał się być zbugowany lub nie rozwiązywał problemu, który rzekomo miał rozwiązywać. 😅
```

File: docs/pl/docs/help-fastapi.md [hunk 10]
```diff
@@ -0,0 +1,265 @@
...
+
+    Dlatego naprawdę ważne jest, abyś faktycznie przeczytał i uruchomił kod oraz dał mi znać w komentarzach, że to zrobiłeś. 🤓
```
Comment (bluefish6) [inline docs/pl/docs/help-fastapi.md:178]:
```suggestion
    Dlatego tak ważne jest, abyś faktycznie przeczytał i uruchomił kod oraz napisał w komentarzu, że to zrobiłeś. 🤓
```

File: docs/pl/docs/help-fastapi.md [hunk 11]
```diff
@@ -0,0 +1,265 @@
...
+
+* Jeśli PR można uprościć w jakiś sposób, możesz o to poprosić, ale nie ma potrzeby być zbyt wybrednym, może być wiele subiektywnych punktów widzenia (a ja też będę miał swoje 🙈), więc lepiej, jeśli skupisz się na podstawowych rzeczach.
```
Comment (bluefish6) [inline docs/pl/docs/help-fastapi.md:180]:
```suggestion
* Jeśli PR można uprościć w jakiś sposób, możesz o to poprosić, ale nie ma potrzeby być zbyt wybrednym, może być wiele subiektywnych punktów widzenia (a ja też będę miał swój 🙈), więc lepiej żebyś skupił się na kluczowych rzeczach.
```

File: docs/pl/docs/help-fastapi.md [hunk 12]
```diff
@@ -0,0 +1,265 @@
...
+
+## Utwórz Pull Request
+
+Możesz [wnieść wkład](contributing.md){.internal-link target=_blank} do kodu źródłowego za pomocą Pull Requestu, na przykład:
+
+* Naprawić literówkę, którą znalazłeś w dokumentacji.
+* Podzielić się artykułem, filmem lub podcastem, który stworzyłeś lub znalazłeś na temat FastAPI, <a href="https://github.com/tiangolo/fastapi/edit/master/docs/en/data/external_links.yml" class="external-link" target="_blank">edytując ten plik</a>.
+    * Upewnij się, że dodajesz swój link na początku odpowiedniej sekcji.
+* Pomóc w [tłumaczeniu dokumentacji](contributing.md#translations){.internal-link target=_blank} na Twój język.
+    * Możesz również pomóc w recenzji tłumaczeń stworzonych przez innych.
```
Comment (bluefish6) [inline docs/pl/docs/help-fastapi.md:202]:
```suggestion
    * Możesz również pomóc w weryfikacji tłumaczeń stworzonych przez innych.
```

File: docs/pl/docs/help-fastapi.md [hunk 13]
```diff
@@ -0,0 +1,265 @@
...
+
+## Pomóż w utrzymaniu FastAPI
+
+Pomóż mi utrzymać **FastAPI**! 🤓
+
+Jest wiele pracy do zrobienia, a w większości przypadków **TY** możesz to zrobić.
+
+Główne zadania, które możesz wykonać teraz to:
+
+* [Pomóc innym z pytaniami na GitHubie](#help-others-with-questions-in-github){.internal-link target=_blank} (zobacz powyższą sekcję).
```
Comment (bluefish6) [inline docs/pl/docs/help-fastapi.md:218]:
```suggestion
* [Pomóc innym z pytaniami na GitHubie](#help-others-with-questions-in-github){.internal-link target=_blank} (zobacz sekcję powyżej).
```

File: docs/pl/docs/help-fastapi.md [hunk 14]
```diff
@@ -0,0 +1,265 @@
...
+* [Oceniać Pull Requesty](#review-pull-requests){.internal-link target=_blank} (zobacz powyższą sekcję).
```
Comment (bluefish6) [inline docs/pl/docs/help-fastapi.md:219]:
```suggestion
* [Oceniać Pull Requesty](#review-pull-requests){.internal-link target=_blank} (zobacz sekcję powyżej).
```

File: docs/pl/docs/help-fastapi.md [hunk 15]
```diff
@@ -0,0 +1,265 @@
...
+
+Te dwie czynności **zajmują najwięcej czasu**. To główna praca związana z utrzymaniem FastAPI.
+
+Jeśli możesz mi w tym pomóc, **pomagasz mi utrzymać FastAPI** i upewnić się, że nadal **rozwija się szybciej i lepiej**. 🚀
```
Comment (bluefish6) [inline docs/pl/docs/help-fastapi.md:223]:
```suggestion
Jeśli możesz mi w tym pomóc, **pomożesz mi utrzymać FastAPI** i zapewnisz że będzie **rozwijać się szybciej i lepiej**. 🚀
```

**Review by bluefish6 — CHANGES_REQUESTED** [review]

File: docs/pl/docs/help-fastapi.md [hunk 16]
```diff
@@ -0,0 +1,265 @@
...
+
+    Używaj czatu tylko do innych ogólnych rozmów.
+
+Istnieje również poprzedni <a href="https://gitter.im/tiangolo/fastapi" class="external-link" target="_blank">czat na Gitter</a>, ale ponieważ nie ma tam kanałów i zaawansowanych funkcji, rozmowy są trudniejsze, dlatego teraz zalecany jest Discord.
+
+### Nie zadawaj pytań na czacie
+
+Miej na uwadze, że ponieważ czaty pozwalają na bardziej "swobodną rozmowę", łatwo jest zadawać pytania, które są zbyt ogólne i trudniejsze do odpowiedzi, więc możesz nie otrzymać odpowiedzi.
+
+Na GitHubie szablon poprowadzi Cię do napisania odpowiedniego pytania, dzięki czemu łatwiej uzyskasz dobrą odpowiedź, a nawet rozwiążesz problem samodzielnie, zanim zapytasz. Pondato na GitHubie mogę się upewnić, że zawsze odpowiadam na wszystko, nawet jeśli zajmuje to trochę czasu. Osobiście nie mogę tego zrobić z systemami czatu. 😅
```
Comment (bluefish6) [inline docs/pl/docs/help-fastapi.md:240]:
```suggestion
Na GitHubie szablon poprowadzi Cię do napisania odpowiedniego pytania, dzięki czemu łatwiej uzyskasz dobrą odpowiedź, a nawet rozwiążesz problem samodzielnie, zanim zapytasz. Ponadto na GitHubie mogę się upewnić, że zawsze odpowiadam na wszystko, nawet jeśli zajmuje to trochę czasu. Osobiście nie mogę tego zrobić z systemami czatu. 😅
```

**Comment by romabozhanovgithub** [comment]
@isulim @bluefish6 Thanks for your review and suggestions!

**Review by bluefish6 — APPROVED** [review]
:+1:

**Comment by tiangolo** [comment]
📝 Docs preview for commit 6788ebc57082506be903ded2970aec2fa64bdb39 at: https://e8b63855.fastapitiangolo.pages.dev

**Comment by romabozhanovgithub** [comment]
@tiangolo Looks like we have 2 approves there

**Comment by tiangolo** [comment]
📝 Docs preview for commit 2d2e7118091427640281bf3c189ac92f482f2d03 at: https://17f45cfb.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
Thanks @romabozhanovgithub! 🙇 

And thanks for the reviews @bluefish6 and @isulim 🍰

**Pull request merged**